tests/end-to-end/tests/accordian.cy.ts

```diff
@@ -0,0 +1,10 @@
+describe('Typerefinery MainComponent', () => {
+  beforeEach(() => {
+    cy.login();
+  });
+  it('renders correctly in preview mode', () => {
+    cy.visit(
+      '/content/typerefinery-showcase/pages/components/content/accodion.html'
```

Comment:
how do we know it renders? can we add tests in here for what you are looking for? and take a screenshot as well?

Reply:
Ok, will test inside the page and try to figure out how to take screenshots.